core/Test/Tasty/Ingredients/ConsoleReporter.hs

```diff
@@ -135,6 +136,7 @@ buildTestOutput opts tree =
           when (not $ null rDesc) $
             (if resultSuccessful result then infoOk else infoFail) $
               printf "%s%s\n" (indent $ level + 1) (formatDesc (level+1) rDesc)
+          maybe (return ()) (\action -> action withConsoleFormat) (printResultDetails result)
```

Comment:
Its likely possible to get a rid of the lambda here.

Reply:
I actually think this maybe should be removed in favor of: https://github.com/mbj/tasty/pull/1/commits/f4a22fbcbf2861adca73bf9f30488e47d64361c3#r337586802

Reply:
@mbj could you replace this with:

```haskell
maybe (return ()) ($ withConsoleFormat) (printResultDetails result)
```

Reply:
I went with removing the maybe so this is redundant.